content/docs/accessibility.md

```diff
@@ -90,6 +85,24 @@ function ListItem({ item }) {
 }
 ```
 
+You may also map a collection of items to an array of fragments as you would any other type of element:
+
+```javascript{6,9}
+function Glossary(props) {
+  return (
+    <dl>
+      {props.items.map(item => (
+        // Fragments should also have a `key` prop when mapping collections
+        <Fragment key={item.id}>
+          <dt>{item.term}</dt>
+          <dd>{item.description}</dd>
+        </Fragment>
+      ))}
+    </dl>
+  );
+}
+```
+
```

Comment:
Nice. But can this be swapped with the example above showing the empty `<></>` way of writing Fragments? I think this would then more logically flow. First showing that you have to put a key on `Fragment` when mapping and THEN if there are no props and your tooling supports it you can use the clean tags. What do you think?

Reply:
Sure! Makes sense.